jekyll/_cci2/server-3-install-prerequisites.adoc

```diff
@@ -45,7 +45,7 @@ Download and install the following software before continuing:
 | {helmversion} or greater
 | Kubernetes Package Management
 
-| KOTS: https://github.com/replicatedhq/kots/releases/download/v1.47.3/kots_darwin_amd64.tar.gz[Mac] or https://github.com/replicatedhq/kots/releases/download/v1.47.3/kots_linux_amd64.tar.gz[Linux].
+| KOTS: https://github.com/replicatedhq/kots/releases/download/v1.65.0/kots_darwin_amd64.tar.gz[Mac] or https://github.com/replicatedhq/kots/releases/download/v1.65.0/kots_linux_amd64.tar.gz[Linux].
 | {kotsversion} *
```

Comment:
where is this value stored? @circleci/docs-disco I wasn't sure how to update it

Reply:
@thomasr8 i'm hoping you might know

Reply:
It's here: https://github.com/circleci/circleci-docs/blob/c2838d9ab54db77ce21c4ab81f7d7224e7713d43/jekyll/_config.yml#L81 and in the section below too (two places for PDF generation - I really need to fix this though!)